sdk/internal/codegen/main.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"log"
+
+	"github.com/opentdf/platform/sdk/internal/codegen/runner"
+)
+
+func main() {
```

Comment:
Aded a comment describing what happens here, and maybe also (re)generate a README.md for the sdkconnect package as well, explaining how to re-generate (or modify, if required)